lib/api/bucketPutNotification.js

```diff
@@ -1,5 +1,4 @@
 const async = require('async');
-
```

Comment:
can you cleanup the added/removed lines that were left over after the latest refactor?